scripts/fetch_service_refs.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+from __future__ import annotations
+
+import argparse
+import json
+import os
+import sys
+import time
+import urllib.error
+import urllib.request
+from typing import Any, Dict, Optional
+
+GO_SERVER_URL = os.environ.get(
+    "GO_SERVER_URL", "http://gocd-server.gocd.svc.cluster.local:8153/go"
```

Comment:
Probably you should rely on environment being set only and not provide a default. The defaults are something which are useful only in our SaaS environments. If we want to use this on Single Tenant or EU regions, these would be wrong.

Reply:
removed it